modules/ivc_champva/spec/requests/ivc_champva/v1/forms/uploads_spec.rb

```diff
@@ -36,6 +36,8 @@
 
       it 'uploads a PDF file to S3' do
         mock_form = double(first_name: 'Veteran', last_name: 'Surname', form_uuid: 'some_uuid')
+        allow(PersistentAttachments::MilitaryRecords).to receive(:find_by)
+          .and_return(double('Record1', created_at: 1.day.ago, id: 'some_uuid', file: double(id: 'file0')))
```

Comment:
Mock added to account for new logic in `supporting_document_ids` that attempts to read `PersistentAttachments::MilitaryRecords`.